### DISCUSSION TITLE
posts/container-fundamentals-learn-container-with-oci-spec/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#37] [Announcements]
# posts/container-fundamentals-learn-container-with-oci-spec/

OCI 规范是事实上的容器标准，已经被大部分容器实现以及容器编排系统所采用，它描述了容器如何被创建和运行，了解 OCI 规范有助于对容器技术建立更本质全面的认知。

https://waynerv.com/posts/container-fundamentals-learn-container-with-oci-spec/

**Comment by peng-yq** [comment]
请问做图是用的什么工具呀？

**Reply by waynerv** [reply]
https://excalidraw.com/

**Comment by fangyuan2015** [comment]
写的挺好的

**Comment by MiiW** [comment]
写的很清晰易懂，赞
